### ISSUE TITLE
Two issues with tracker since yesterday

### ISSUE THREAD

**Issue by MCHAWKING-cpu** [ShootMe/FallGuysStats#98]
Hi

I didn't use the tracker on the patch day, but on Friday I started using it again and I noticed the following issues (confirmed with other people I play with using the tracker)

1) Ping not dynamically updating. It seems the tracker checks the ping once on startup and then does not check it again, leaving the same reading the entire time it is running.

2) Some shows are not being recorded after an elimination if you leave before the show entirely ends (ie. a winner gets a crown). This might have to do with the fact that the reward screen seems to often be skipped now and you just get a black screen saying "calculating rewards" or something to that effect.

**Comment by wolfchild59** [comment]
Have you noticed if your Slimescraper stats are being recorded at all? My tracker says I'm on the latest version, and I see Slimescraper in the listings, but it's not logging anything for it. It is capturing the data for Button Bashers though.

**Comment by MCHAWKING-cpu** [comment]
> 
> 
> Have you noticed if your Slimescraper stats are being recorded at all? My tracker says I'm on the latest version, and I see Slimescraper in the listings, but it's not logging anything for it. It is capturing the data for Button Bashers though.

Yes I have slimescraper stats recorded, but again it might not have saved every run of it I have done because some shows it does not save. But I don't see any issue with slimescraper in particular.
